ArduPlane/quadplane.cpp

```diff
@@ -1116,6 +1116,16 @@ void QuadPlane::control_qacro(void)
     }
 }
 
+void QuadPlane::relax_attitude_control(){
+    // if not a vectored tailsitter completely disable attitude control
+    uint8_t chan;
+    bool vectored_tailsitter = SRV_Channels::find_channel(SRV_Channel::k_tiltMotorLeft, chan) &&
+                               SRV_Channels::find_channel(SRV_Channel::k_tiltMotorLeft, chan);
+    if (!is_tailsitter() || !vectored_tailsitter) {
+        attitude_control->relax_roll_yaw_controllers();
+    }
```

Comment:
We have stopped relaxing at all on most things?
```suggestion
    }
    if (is_tailsitter() && vectored_tailsitter) {
        // don't relax pitch on vectored tailsitters
        attitude_control->relax_roll_yaw_controllers();
    } else {
        attitude_control->relax_attitude_controllers();
    }
```

Reply:
you can tell I forgot to test this after making a massive change...